### PR TITLE
Feature/view organization profile

### PR DESCRIPTION
**Pull request by Chimaobi-Johnson**
### What does this pull request aim to achieve? 💯

1. Completed UI for view organization page and Implemented functionality
2. Implemented functionality for edit organization profile
3. Implemented functionality for edit organization account

UNRESOLVED ISSUES
1. there is currently no error handling code for the backend, so there is no way of seeing an error on the UI
2. the location input in the edit organization account does not display a default value
3. there is no way to view organizations you created (Didnt see a desktop page for this on figma )
4. there is no backend query to see organizations created by the logged in user (I didnt have time to look into this)
5. there is currently no feed in the organization profile.
6. there are currently no svg icons for appstore and googlestore in the organization profile.
7. the location input on create organization profile only accepts address with zipcode

_Please be concise and link any related issue(s):_

### 🚨Before submitting this pull request🚨:

_Please do **NOT** submit this PR if you have not done the following:_

1. I have checked that no one else is working on similar changes.
2. I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
3. The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
4. This branch is rebased/merged with the **latest master**.
5. There are no merge conflicts.
6. There are no console warnings and errors.
7. On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: backend/lib/endpoints/organizations.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ async function routes(app) {
           organization[key] = req.body[key];
         }
       });
-      return organization.save();
+      return organization.save();c
```
[the remark this answers is not in the document]
Comment (Chimaobi-Johnson) [reply backend/lib/endpoints/organizations.js:75]: oh thats probably a mistake

[the remark this answers is not in the document]
Comment (idjevm) [reply backend/lib/endpoints/organizations.js:75]: No worries. Besides this, it is looking great!

**Comment by joshmorel** [comment]
@Chimaobi-Johnson thanks for this work! From what I understand further work is to be undertaken by others. 

@robinv85 `ownerId` is an expected property but the front-end is not passing it, should we just do something like this. I can add commits here (despite the title - this also covers getting organization CRUD to fully work, no?) 

```javascript
    async (req) => {
      const { userId: ownerId } = req;
      return new Organization({ ...req.body, ownerId }).save();
    },
```

File: backend/lib/endpoints/schema/organizations.js [hunk 2]
```diff
@@ -3,22 +3,26 @@ const { strictSchema } = require("./utils");
...
   language: S.string(),
   location: locationSchema,
   name: S.string(),
-  needs: S.array().maxItems(10).items(S.string()),
+  needs: S.object()
+    .prop("volunteers", S.boolean().required().default(false))
+    .prop("donations", S.boolean().required().default(false))
+    .prop("staff", S.boolean().required().default(false))
+    .prop("other", S.boolean().required().default(false)),
   ownerId: S.string(),
```
Comment (robinv85) [inline backend/lib/endpoints/schema/organizations.js:18]: this should be required?

File: backend/lib/endpoints/schema/organizations.js [hunk 3]
```diff
@@ -3,22 +3,26 @@ const { strictSchema } = require("./utils");
...
 
 const organization = {
-  about: S.string(),
+  about: S.string().maxLength(100),
   email: S.string().format("email"),
   global: S.boolean(),
   industry: S.string(),
   language: S.string(),
   location: locationSchema,
   name: S.string(),
```
Comment (robinv85) [inline backend/lib/endpoints/schema/organizations.js:12]: this should also be required?

File: backend/lib/endpoints/schema/organizations.js [hunk 4]
```diff
@@ -3,22 +3,26 @@ const { strictSchema } = require("./utils");
...
+    .prop("other", S.boolean().required().default(false)),
   ownerId: S.string(),
   type: S.string(),
-  url: S.object()
-    .prop("appStore", S.string().format("url"))
-    .prop("linkedin", S.string().format("url"))
-    .prop("playStore", S.string().format("url"))
-    .prop("twitter", S.string().format("url"))
-    .prop("website", S.string().format("url")),
+  urls: S.object()
```
Comment (robinv85) [inline backend/lib/endpoints/schema/organizations.js:20]: we could probably reuse the `urls` from the individual users, at least some of the validation rules
but it also seemed like the linkedIn URL wasn't correct

**Comment by robinv85** [comment]
@Chimaobi-Johnson can you fix the merge conflict and merge it? Probably some things slightly changed like `AddressInput` to  `LocationInput` etc. on latest master

**Comment by joshmorel** [comment]
Merge conflict is fixed for locationInput as well as a few other issues. The biggest item to work on next is: 

> there is no way to view organizations you created (Didnt see a desktop page for this on figma

@vikjoshi will create a new issue for this.

**Pull request merged**